examples/generation/lm.ipynb

```diff
@@ -120,6 +122,27 @@
     "        embeddings = self.embedder(input_tokens)\n",
     "        rnn_hidden = self.rnn(embeddings, mask)\n",
     "        out_logits = self.hidden2out(rnn_hidden)\n",
+    "\n",
+    "        \"\"\"\n",
+    "        THIS IS LIKELY NOT HOW I SHOULD FIX THIS BUT IT WAS THE BEST\n",
+    "        I COULD DO TO GET THIS WORKING\n",
+    "\n",
+    "        At this stage, `output_tokens` looks like this (module the specific token indices):\n",
+    "\n",
+    "        {'tokens': {'tokens': tensor([[16, 45,  5,  ...,  0,  0,  0],\n",
+    "                [51, 56, 48,  ...,  0,  0,  0],\n",
+    "                [44, 54,  2,  ...,  0,  0,  0],\n",
+    "                ...,\n",
+    "                [14, 54,  7,  ...,  0,  0,  0],\n",
+    "                [10, 48, 22,  ...,  0,  0,  0],\n",
+    "                [51, 36, 56,  ..., 58,  0,  0]])}}\n",
+    "\n",
+    "        which seems like it's being double indexed somehow.\n",
+    "\n",
+    "        Thus, calling output_tokens = output_tokens[\"tokens\"] to unnest `tokens`\n",
+    "        resolves this in an unideal way.\n",
+    "        \"\"\"\n",
+    "        output_tokens = output_tokens[\"tokens\"]\n",
```

Comment:
I couldn't quite figure out the correct way to wrap the data loaders from AllenNLP to avoid this nesting of the `output_tokens`.